homeassistant/components/trafikverket_camera/entity.py

```diff
@@ -0,0 +1,62 @@
+"""Base entity for Trafikverket Camera."""
+from __future__ import annotations
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import TVDataUpdateCoordinator
+
+
+class TrafikverketCameraEntity(CoordinatorEntity[TVDataUpdateCoordinator]):
```

Comment:
This feels a bit unneeded?
Making a base class for `sensor` and `binary_sensor` yes but for the camera not so much (as it saves no code)

Reply:
It saves the device info duplication, or at least that was the intention

Reply:
Yeah. Let's leave it with the other changes now requested